src/tests/ftest/util/server_utils.py

```diff
@@ -196,7 +196,8 @@ def __init__(self):
                  "CRT_CTX_SHARE_ADDR=0",
                  "CRT_TIMEOUT=30",
                  "FI_SOCKETS_MAX_CONN_RETRY=1",
-                 "FI_SOCKETS_CONN_TIMEOUT=2000"]
+                 "FI_SOCKETS_CONN_TIMEOUT=2000",
+                 "DD_MASK=mgmt,io,md,epc,rebuild"]
```

Comment:
so daos_default is io,md,pl, rebuild,sec,csum

Is there a reason for the change here?   Just trying to understand where the set came from?

Reply:
@jolivier23 checkout the convo in the ticket here: https://jira.hpdd.intel.com/browse/DAOS-3280

Reply:
@jolivier23, do you think I should modify the set to add pl?

Reply:
@amjustin13 that is fine.   But I think rather than setting it in the environment, we should just change it in the source.   By default DD_MASK is set to daos_default which is defined by this group of flags in src/common/debug.c

45 /* debug bit groups */
 46 #define DB_GRP1 (DB_IO | DB_MD | DB_PL | DB_REBUILD | DB_SEC | DB_CSUM)

If we want to change the default, we should just change the default